src/components/Header.jsx

```diff
@@ -23,6 +27,13 @@ function Header() {
               >
                   Vans
               </NavLink>
+              <Link to="login" className="login-link">
+                <img 
+                    src="/avatar-icon.png" 
+                    className="login-icon"
```

Comment:
One of the rules you need to follow while using the `img` tag is to add an `alt` attribute to it. This attribute describes the image and is very useful when the image does not load. The `alt` attribute will be displayed to help the user understand what the image was.